app/src/main/java/com/portfolio/tasky/entry/fragments/RegistrationFragment.kt

```diff
@@ -40,7 +42,15 @@ class RegistrationFragment : Fragment(), FragmentInflater by FragmentInflaterImp
 
         setObservers()
         viewBinding.btnReg.setOnClickListener {
-            startLoginFragment()
+            val name = viewBinding.etName.subLayout.etInput.text.toString()
+            val email = viewBinding.etEmail.subLayout.etInput.text.toString()
+            val password = viewBinding.etPassword.subLayout.etInput.text.toString()
+
+            viewModel.makeRegistrationCall(RegisterRequest(name, email, password)).observe(viewLifecycleOwner){
+                if(it){
+                    startLoginFragment()
+                }
```

Comment:
If registration fails, we can show an error or something

Reply:
On failure of registration we already show error from TaskyStatusImpl. You can check the video how it shows error on entering wrong login password, same happens for registration.  This is just a dummy functionality.

Reply:
Sounds good